### DISCUSSION TITLE
Why some queries don't contain DNS server details field?

### DISCUSSION THREAD

**Discussion by uNitrous** [AdguardTeam/AdGuardHome#2769] [Q&A] [answered]
![Screenshot from 2021-02-27 02-00-31](https://user-images.githubusercontent.com/63360418/109343676-4bfe7580-78a0-11eb-9cca-0b7981db226a.png)
![Screenshot from 2021-02-27 02-01-20](https://user-images.githubusercontent.com/63360418/109343686-4dc83900-78a0-11eb-985b-0ad303722a52.png)

**Comment by ameshkov** [comment, marked as answer]
The one without server info is served from the DNS cache
